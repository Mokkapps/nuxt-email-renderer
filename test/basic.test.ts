import { resolve } from 'node:path'
import { describe, it, expect } from 'vitest'
import { setup, $fetch } from '@nuxt/test-utils/e2e'

describe.skip('ssr', async () => {
  await setup({
    rootDir: resolve(__dirname, './fixtures/basic'),
  })

  it('renders the index page', async () => {
    // Get response to a server-rendered page with `$fetch`.
    const html = await $fetch('/')
    expect(html).toContain('<div>basic</div>')
  })
})
