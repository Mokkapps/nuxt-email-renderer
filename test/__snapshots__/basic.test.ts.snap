// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`E2E > API > returns the rendered email as HTML 1`] = `
"<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html lang="en" dir="ltr"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="x-apple-disable-message-reformatting"></head> <div style="display:none;overflow:hidden;line-height:1px;opacity:0;max-height:0;max-width:0;">Welcome to our platform! Get started with your account today.<div> ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿</div></div> <body style=""><table border="0" width="100%" cellpadding="0" cellspacing="0" role="presentation" align="center"><tbody><tr><td style="background-color:#ffffff;font-family:-apple-system, BlinkMacSystemFont, &#39;Segoe UI&#39;, &#39;Roboto&#39;, &#39;Oxygen&#39;, &#39;Ubuntu&#39;, &#39;Cantarell&#39;, &#39;Fira Sans&#39;, &#39;Droid Sans&#39;, &#39;Helvetica Neue&#39;, sans-serif;"><table align="center" width="100%" role="presentation" cellspacing="0" cellpadding="0" border="0" style="max-width:600px;background-color:#ffffff;border-radius:8px;border:1px solid #e5e7eb;margin:40px auto;padding:20px;"><tbody><tr style="width: 100%"><td> <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;"><tbody><tr><td><img style="display:block;outline:none;border:none;text-decoration:none;margin:0 auto;" src="https://images.unsplash.com/photo-1560472354-b33ff0c44a43?w=200&amp;h=60&amp;fit=crop&amp;crop=center" width="200" height="60" alt="Company Logo"></td></tr></tbody></table>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;"><tbody><tr><td><h1 style="color:#1f2937;font-size:32px;font-weight:bold;text-align:center;margin:0 0 20px 0;">Welcome to our platform! 🎉</h1> <p style="font-size:14px;line-height:24px;margin:0 0 16px 0;color:#374151;font-size:16px;line-height:1.6;">
            Hi there,
          </p> <p style="font-size:14px;line-height:24px;margin:0 0 16px 0;color:#374151;font-size:16px;line-height:1.6;">
            Welcome to our platform! We&#39;re thrilled to have you join our community of innovators and creators.
            Your journey to amazing possibilities starts right here.
          </p> <p style="font-size:14px;line-height:24px;margin:0 0 24px 0;color:#374151;font-size:16px;line-height:1.6;">
            To get you started, here are a few things you can do:
          </p>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="background-color:#f9fafb;border-radius:8px;padding:20px;margin:20px 0;"><tbody><tr><td><p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#374151;font-size:16px;line-height:1.6;">
              ✨ <strong>Complete your profile</strong> - Add your details to personalize your experience
            </p> <p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#374151;font-size:16px;line-height:1.6;">
              🚀 <strong>Explore features</strong> - Discover all the powerful tools at your disposal
            </p> <p style="font-size:14px;line-height:24px;margin:0;color:#374151;font-size:16px;line-height:1.6;">
              💬 <strong>Join the community</strong> - Connect with other users and share experiences
            </p></td></tr></tbody></table></td></tr></tbody></table>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="text-align:center;padding:20px 0;"><tbody><tr><td><a style="line-height:100%;text-decoration:none;display:inline-block;max-width:100%;padding:16px 32px;background-color:#3b82f6;color:#ffffff;border-radius:8px;text-decoration:none;font-weight:600;font-size:16px;" href="https://example.com/dashboard"><span></span><span style="max-width:100%;display:inline-block;line-height:120%;mso-padding-alt:0px;mso-text-raise:0;">
            Get Started Now
          </span><span></span></a></td></tr></tbody></table>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;border-top:1px solid #e5e7eb;"><tbody><tr><td><p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#6b7280;font-size:14px;line-height:1.6;text-align:center;">
            Need help getting started? We&#39;re here for you!
          </p> <p style="font-size:14px;line-height:24px;margin:0;color:#6b7280;font-size:14px;line-height:1.6;text-align:center;">
            Visit our <a style="color:#3b82f6;text-decoration:none;" href="https://example.com/help" target="_blank">
              Help Center
            </a> or reply to this email.
          </p></td></tr></tbody></table>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;border-top:1px solid #e5e7eb;"><tbody><tr><td><p style="font-size:14px;line-height:24px;margin:0;color:#6b7280;font-size:12px;text-align:center;">
            © 2024 Your Company. All rights reserved.<br>
            123 Business Street, City, State 12345
          </p></td></tr></tbody></table></td></tr></tbody></table></td></tr></tbody></table></body></html>"
`;

exports[`E2E > API > returns the rendered email as formatted HTML 1`] = `
"<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en" dir="ltr">

  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta name="x-apple-disable-message-reformatting">
  </head>
  <div style="display:none;overflow:hidden;line-height:1px;opacity:0;max-height:0;max-width:0;">Welcome to our platform! Get started with your account today.<div> ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿</div>
  </div>

  <body style="">
    <table border="0" width="100%" cellpadding="0" cellspacing="0" role="presentation" align="center">
      <tbody>
        <tr>
          <td style="background-color:#ffffff;font-family:-apple-system, BlinkMacSystemFont, &#39;Segoe UI&#39;, &#39;Roboto&#39;, &#39;Oxygen&#39;, &#39;Ubuntu&#39;, &#39;Cantarell&#39;, &#39;Fira Sans&#39;, &#39;Droid Sans&#39;, &#39;Helvetica Neue&#39;, sans-serif;">
            <table align="center" width="100%" role="presentation" cellspacing="0" cellpadding="0" border="0" style="max-width:600px;background-color:#ffffff;border-radius:8px;border:1px solid #e5e7eb;margin:40px auto;padding:20px;">
              <tbody>
                <tr style="width: 100%">
                  <td>
                    <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;">
                      <tbody>
                        <tr>
                          <td><img style="display:block;outline:none;border:none;text-decoration:none;margin:0 auto;" src="https://images.unsplash.com/photo-1560472354-b33ff0c44a43?w=200&amp;h=60&amp;fit=crop&amp;crop=center" width="200" height="60" alt="Company Logo"></td>
                        </tr>
                      </tbody>
                    </table>
                    <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;">
                      <tbody>
                        <tr>
                          <td>
                            <h1 style="color:#1f2937;font-size:32px;font-weight:bold;text-align:center;margin:0 0 20px 0;">Welcome to our platform! 🎉</h1>
                            <p style="font-size:14px;line-height:24px;margin:0 0 16px 0;color:#374151;font-size:16px;line-height:1.6;">
                              Hi there,
                            </p>
                            <p style="font-size:14px;line-height:24px;margin:0 0 16px 0;color:#374151;font-size:16px;line-height:1.6;">
                              Welcome to our platform! We&#39;re thrilled to have you join our community of innovators and creators.
                              Your journey to amazing possibilities starts right here.
                            </p>
                            <p style="font-size:14px;line-height:24px;margin:0 0 24px 0;color:#374151;font-size:16px;line-height:1.6;">
                              To get you started, here are a few things you can do:
                            </p>
                            <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="background-color:#f9fafb;border-radius:8px;padding:20px;margin:20px 0;">
                              <tbody>
                                <tr>
                                  <td>
                                    <p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#374151;font-size:16px;line-height:1.6;">
                                      ✨ <strong>Complete your profile</strong> - Add your details to personalize your experience
                                    </p>
                                    <p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#374151;font-size:16px;line-height:1.6;">
                                      🚀 <strong>Explore features</strong> - Discover all the powerful tools at your disposal
                                    </p>
                                    <p style="font-size:14px;line-height:24px;margin:0;color:#374151;font-size:16px;line-height:1.6;">
                                      💬 <strong>Join the community</strong> - Connect with other users and share experiences
                                    </p>
                                  </td>
                                </tr>
                              </tbody>
                            </table>
                          </td>
                        </tr>
                      </tbody>
                    </table>
                    <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="text-align:center;padding:20px 0;">
                      <tbody>
                        <tr>
                          <td><a style="line-height:100%;text-decoration:none;display:inline-block;max-width:100%;padding:16px 32px;background-color:#3b82f6;color:#ffffff;border-radius:8px;text-decoration:none;font-weight:600;font-size:16px;" href="https://example.com/dashboard"><span></span><span style="max-width:100%;display:inline-block;line-height:120%;mso-padding-alt:0px;mso-text-raise:0;">
            Get Started Now
          </span><span></span></a></td>
                        </tr>
                      </tbody>
                    </table>
                    <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;border-top:1px solid #e5e7eb;">
                      <tbody>
                        <tr>
                          <td>
                            <p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#6b7280;font-size:14px;line-height:1.6;text-align:center;">
                              Need help getting started? We&#39;re here for you!
                            </p>
                            <p style="font-size:14px;line-height:24px;margin:0;color:#6b7280;font-size:14px;line-height:1.6;text-align:center;">
                              Visit our <a style="color:#3b82f6;text-decoration:none;" href="https://example.com/help" target="_blank">
                                Help Center
                              </a> or reply to this email.
                            </p>
                          </td>
                        </tr>
                      </tbody>
                    </table>
                    <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;border-top:1px solid #e5e7eb;">
                      <tbody>
                        <tr>
                          <td>
                            <p style="font-size:14px;line-height:24px;margin:0;color:#6b7280;font-size:12px;text-align:center;">
                              © 2024 Your Company. All rights reserved.<br>
                              123 Business Street, City, State 12345
                            </p>
                          </td>
                        </tr>
                      </tbody>
                    </table>
                  </td>
                </tr>
              </tbody>
            </table>
          </td>
        </tr>
      </tbody>
    </table>
  </body>

</html>"
`;

exports[`E2E > API > returns the rendered email as plain text 1`] = `
"WELCOME TO OUR PLATFORM! 🎉

Hi there,

Welcome to our platform! We're thrilled to have you join our community of
innovators and creators. Your journey to amazing possibilities starts right
here.

To get you started, here are a few things you can do:

✨ Complete your profile - Add your details to personalize your experience

🚀 Explore features - Discover all the powerful tools at your disposal

💬 Join the community - Connect with other users and share experiences

Get Started Now https://example.com/dashboard

Need help getting started? We're here for you!

Visit our Help Center https://example.com/help or reply to this email.

© 2024 Your Company. All rights reserved.
123 Business Street, City, State 12345"
`;

exports[`E2E > API > returns the rendered email with props as HTML 1`] = `
"<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html lang="en" dir="ltr"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="x-apple-disable-message-reformatting"></head> <div style="display:none;overflow:hidden;line-height:1px;opacity:0;max-height:0;max-width:0;">Welcome to our platform! Get started with your account today.<div> ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿ ‌​‍‎‏﻿</div></div> <body style=""><table border="0" width="100%" cellpadding="0" cellspacing="0" role="presentation" align="center"><tbody><tr><td style="background-color:#ffffff;font-family:-apple-system, BlinkMacSystemFont, &#39;Segoe UI&#39;, &#39;Roboto&#39;, &#39;Oxygen&#39;, &#39;Ubuntu&#39;, &#39;Cantarell&#39;, &#39;Fira Sans&#39;, &#39;Droid Sans&#39;, &#39;Helvetica Neue&#39;, sans-serif;"><table align="center" width="100%" role="presentation" cellspacing="0" cellpadding="0" border="0" style="max-width:600px;background-color:#ffffff;border-radius:8px;border:1px solid #e5e7eb;margin:40px auto;padding:20px;"><tbody><tr style="width: 100%"><td> <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;"><tbody><tr><td><img style="display:block;outline:none;border:none;text-decoration:none;margin:0 auto;" src="https://images.unsplash.com/photo-1560472354-b33ff0c44a43?w=200&amp;h=60&amp;fit=crop&amp;crop=center" width="200" height="60" alt="Company Logo"></td></tr></tbody></table>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;"><tbody><tr><td><h1 style="color:#1f2937;font-size:32px;font-weight:bold;text-align:center;margin:0 0 20px 0;">Test Heading</h1> <p style="font-size:14px;line-height:24px;margin:0 0 16px 0;color:#374151;font-size:16px;line-height:1.6;">
            Hi there,
          </p> <p style="font-size:14px;line-height:24px;margin:0 0 16px 0;color:#374151;font-size:16px;line-height:1.6;">
            Welcome to our platform! We&#39;re thrilled to have you join our community of innovators and creators.
            Your journey to amazing possibilities starts right here.
          </p> <p style="font-size:14px;line-height:24px;margin:0 0 24px 0;color:#374151;font-size:16px;line-height:1.6;">
            To get you started, here are a few things you can do:
          </p>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="background-color:#f9fafb;border-radius:8px;padding:20px;margin:20px 0;"><tbody><tr><td><p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#374151;font-size:16px;line-height:1.6;">
              ✨ <strong>Complete your profile</strong> - Add your details to personalize your experience
            </p> <p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#374151;font-size:16px;line-height:1.6;">
              🚀 <strong>Explore features</strong> - Discover all the powerful tools at your disposal
            </p> <p style="font-size:14px;line-height:24px;margin:0;color:#374151;font-size:16px;line-height:1.6;">
              💬 <strong>Join the community</strong> - Connect with other users and share experiences
            </p></td></tr></tbody></table></td></tr></tbody></table>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="text-align:center;padding:20px 0;"><tbody><tr><td><a style="line-height:100%;text-decoration:none;display:inline-block;max-width:100%;padding:16px 32px;background-color:#3b82f6;color:#ffffff;border-radius:8px;text-decoration:none;font-weight:600;font-size:16px;" href="https://example.com/dashboard"><span></span><span style="max-width:100%;display:inline-block;line-height:120%;mso-padding-alt:0px;mso-text-raise:0;">
            Get Started Now
          </span><span></span></a></td></tr></tbody></table>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;border-top:1px solid #e5e7eb;"><tbody><tr><td><p style="font-size:14px;line-height:24px;margin:0 0 12px 0;color:#6b7280;font-size:14px;line-height:1.6;text-align:center;">
            Need help getting started? We&#39;re here for you!
          </p> <p style="font-size:14px;line-height:24px;margin:0;color:#6b7280;font-size:14px;line-height:1.6;text-align:center;">
            Visit our <a style="color:#3b82f6;text-decoration:none;" href="https://example.com/help" target="_blank">
              Help Center
            </a> or reply to this email.
          </p></td></tr></tbody></table>  <table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="padding:20px 0;border-top:1px solid #e5e7eb;"><tbody><tr><td><p style="font-size:14px;line-height:24px;margin:0;color:#6b7280;font-size:12px;text-align:center;">
            © 2024 Your Company. All rights reserved.<br>
            123 Business Street, City, State 12345
          </p></td></tr></tbody></table></td></tr></tbody></table></td></tr></tbody></table></body></html>"
`;
